tdrs-backend/tdpservice/data_files/admin.py

```diff
@@ -3,19 +3,94 @@
 
 from ..core.utils import ReadOnlyAdminMixin
 from .models import DataFile, LegacyFileTransfer
+from tdpservice.parsers.models import DataFileSummary, ParserError
+from django.conf import settings
+from django.utils.html import format_html
 
+DOMAIN = settings.FRONTEND_BASE_URL
+
+class DataFileSummaryStatusFilter(admin.SimpleListFilter):
+    """Admin class filter for file status (accepted, rejected) for datafile model."""
+
+    title = 'status'
+    parameter_name = 'status'
+
+    def lookups(self, request, model_admin):
+        """Return a list of tuples."""
+        return [
+            ('Accepted', 'Accepted'),
+            ('Rejected', 'Rejected'),
```

Comment:
we have a couple more statuses than just Accepted and Rejected, could they be added? also rather than create a custom filter class, you could add `'datafilesummary__status'` to the `DataFileAdmin`'s `list_filter` - i tested that out quickly and it correctly ports all the `DataFileSummary.Status` options without needing to hard code anything
<img width="313" alt="image" src="https://github.com/raft-tech/TANF-app/assets/111305129/e60b8db9-c9c6-434d-9970-8fc24eabcecb">
<img width="323" alt="image" src="https://github.com/raft-tech/TANF-app/assets/111305129/c2fff3ba-838a-45c0-9617-e7fa0a0b6eab">